src/renderers/dom/client/eventPlugins/ChangeEventPlugin.js

```diff
@@ -45,6 +47,35 @@ var eventTypes = {
   },
 };
 
+function getInstIfValueChanged(targetInst, target) {
+  var value = getInputValue(target);
+  var lastValue = target[LAST_VALUE_KEY];
+
+  if (!target.hasOwnProperty(LAST_VALUE_KEY) || value !== lastValue) {
+    target[LAST_VALUE_KEY] = value;
```

Comment:
actually...it seems likes `node.value = 'foo'` doesn't trigger a change...which I could have sworn was not the case. Maybe there isn't any issue (from my perspective)